dimensiondata/src/main/java/org/jclouds/dimensiondata/cloudcontrol/DimensionDataCloudControlProviderMetadata.java

```diff
@@ -42,12 +48,38 @@ public DimensionDataCloudControlProviderMetadata() {
       super(builder());
    }
 
-   public DimensionDataCloudControlProviderMetadata(Builder builder) {
+   public DimensionDataCloudControlProviderMetadata(final Builder builder) {
       super(builder);
    }
 
    public static Properties defaultProperties() {
       Properties properties = DimensionDataCloudControlApiMetadata.defaultProperties();
+      properties.setProperty(PROPERTY_REGIONS, "na,eu,au,mea,ap,canada");
+      properties.setProperty(PROPERTY_ISO3166_CODES, "NA,EU,AU,AF,AP,CA");
```

Comment:
Since the regions expand several countries I think we can't really put a proper ISO code for them (like "US", "DE", etc). I'd remove the ISO code properties for the regions and just leave them for the zones. Makes sense?

Reply:
Makes sense I can do this update